llx/code.go

```diff
@@ -399,7 +399,25 @@ func (l *CodeV2) entrypoint2assessment(bundle *CodeBundle, ref uint64, lookup fu
 
 		if !truthy {
 			listRef := chunk.Function.Binding
-			list, ok := lookup(code.Checksums[listRef])
+			for i := range code.Blocks {
+				if code.Blocks[i].Entrypoints[0] != ref {
+					continue
+				}
+				for j := len(code.Blocks[i].Datapoints) - 1; j >= 0; j-- {
+					// skip the resource ids datapoint
+					if code.Blocks[i].Datapoints[j] == listRef {
+						continue
+					}
+					chunk := code.Chunk(code.Blocks[i].Datapoints[j])
+					// this contains the default values
+					if chunk.Function.Binding == listRef {
+						listRef = code.Blocks[i].Datapoints[j]
+						break
```

Comment:
does this break only the inner loop? should it break the outer loop?

Reply:
Good catch. It can actually break the outer loop. Changed it.